internal/flagv1/flag_priv_test.go

```diff
@@ -1,10 +1,11 @@
 package flagv1
 
 import (
-	"github.com/stretchr/testify/assert"
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
Can we keep the import as before, there is no change in them just a different order.